invenio_pidstore/models.py

```diff
@@ -55,12 +55,12 @@
 from datetime import datetime
 
 from sqlalchemy.exc import SQLAlchemyError
+from invenio_db import db, InvenioDB
 
-from invenio_base.globals import cfg
-from invenio_ext.sqlalchemy import db
 from invenio_utils.text import to_unicode
```

Comment:
I will have to rely on invenio_utils, is this OK or should I find equivalent method in some other available libs ? @lnielsen